CHANGELOG.md

```diff
@@ -3,3 +3,7 @@
 
 ## [1.0.1] - 2025-05-20
 - Fix wrong provider name in composer.json
+
+## [1.0.2] - 2025-07-31
+- Added config file
```

Comment:
- chore: Renamed configuration file #3 


By the way you did not add a config file, you renamed it.